gpu_bdb/cluster_configuration/cluster-startup-slurm.sh

```diff
@@ -75,7 +75,7 @@ if [ "$ROLE" = "SCHEDULER" ]; then
   fi
   
   if [ "$CLUSTER_MODE" = "IB" ]; then
-     DASK_RMM__POOL_SIZE=1GB CUDA_VISIBLE_DEVICES='0' DASK_UCX__CUDA_COPY=True DASK_UCX__TCP=True DASK_UCX__NVLINK=True DASK_UCX__INFINIBAND=True DASK_UCX__RDMACM=False UCX_NET_DEVICES=mlx5_1:1 nohup dask-scheduler --dashboard-address 8787 --protocol ucx --interface ibp18s0 --scheduler-file $SCHEDULER_FILE > $LOGDIR/$HOSTNAME-scheduler.log 2>&1 &
+     DASK_RMM__POOL_SIZE=1GB CUDA_VISIBLE_DEVICES='0' DASK_DISTRIBUTED__COMM__UCX__CREATE_CUDA_CONTEXT=True nohup dask-scheduler --dashboard-address 8787 --protocol ucx --interface ibp18s0 --scheduler-file $SCHEDULER_FILE > $LOGDIR/$HOSTNAME-scheduler.log 2>&1 &
```

Comment:
Do we also need `UCX_MEMTYPE_REG_WHOLE_ALLOC_TYPES` and `UCX_MAX_RNDV_RAILS` with ucx 1.13 or are these enabled by default?

Reply:
[`UCX_MEMTYPE_REG_WHOLE_ALLOC_TYPES=cuda` is the default on UCX >= 1.12.0](https://ucx-py.readthedocs.io/en/latest/configuration.html#ucx-memtype-reg-whole-alloc-types), and [`UCX_MAX_RNDV_RAILS=1` is the default in UCX-Py since RAPIDS 22.04 or so](https://github.com/rapidsai/ucx-py/blob/branch-0.28/ucp/__init__.py#L84-L86).